src/Essentials/src/AppActions/AppActions.shared.cs

```diff
@@ -8,12 +8,12 @@ namespace Microsoft.Maui.ApplicationModel
 	/// <summary>
 	/// The AppActions API lets you create and respond to app shortcuts from the app icon.
 	/// </summary>
-	public interface IAppActions
+	public interface IAppActions : Microsoft.Maui.Devices.IDeviceCapability
```

Comment:
Please tell me why the full name was used instead of just `IDeviceCapability`. Don't change this, just tell me.